src/masoniteorm/schema/Column.py

```diff
@@ -61,16 +62,18 @@ def after(self, after):
         self.after = after
         return self
 
-    def default(self, value):
+    def default(self, value, raw=False):
```

Comment:
So this is used like:

```python
table.string("name").default("something", raw=True)
```

?

Can we make a new method like:

```python
table.string("name").default_raw("something")
```

This would just call the `default('something', raw=True)` under the hood

Reply:
Also can we add a test for this schema? Can do it inside one of the test_postgres_alter_schema.py files

Reply:
Yep NP 
Will work on these this afternoon

Reply:
@josephmancuso 

Added the default_raw() method as suggested
Added variations across all grammars tests  for both .default("value", True) and .default_raw("value")